controller_interface/src/controller_interface_base.cpp

```diff
@@ -130,4 +130,6 @@ std::shared_ptr<rclcpp_lifecycle::LifecycleNode> ControllerInterfaceBase::get_no
 
 unsigned int ControllerInterfaceBase::get_update_rate() const { return update_rate_; }
 
+bool ControllerInterfaceBase::is_async() const { return is_async_; }
```

Comment:
I think we have to update the header file too. Isn't it?

Reply:
It was updated in the previous PR, I just forgot to include the function's definition.